apps/prairielearn/src/pages/instructorStudents/instructorStudents.html.tsx

```diff
@@ -294,85 +263,70 @@ function StudentsCard({
           {lastInvitation.pending_uid} was invited successfully.
         </Alert>
       )}
-      <div class="card d-flex flex-column h-100">
-        <div class="card-header bg-primary text-white">
-          <div class="d-flex align-items-center justify-content-between gap-2">
-            <div>Students</div>
-            <div class="d-flex gap-2">
-              <DownloadButton
-                course={course}
-                courseInstance={courseInstance}
-                students={students}
-                table={table}
+      <TanstackTableCard
```

Comment:
It's very likely that we'll want to decouple the table and filters from the "card" UI, e.g. for course instance extensions (we probably won't have a card there). I'm fine to defer that until it's needed, we don't have to get the abstractions perfectly correct just yet.